lib/compat/wordpress-6.1/class-wp-theme-json-data-gutenberg.php

```diff
@@ -24,6 +24,27 @@ class WP_Theme_JSON_Data_Gutenberg {
 	 */
 	private $origin = '';
 
+	/**
+	 * Container for keep track of registered blocks.
+	 *
+	 * @since 6.1.0
+	 * @var array
+	 */
+	protected static $blocks_cache = array(
+		'core'   => array(),
+		'blocks' => array(),
+		'theme'  => array(),
+		'user'   => array(),
+	);
+
+	/**
+	 * Container for data coming from the blocks.
+	 *
+	 * @since 6.1.0
+	 * @var WP_Theme_JSON
+	 */
+	protected static $blocks = null;
```

Comment:
These changes were not part of Core commit https://core.trac.wordpress.org/changeset/54493. I'll revert them.